test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -368,7 +368,7 @@ static address reserve_multiple(int num_stripes, size_t stripe_len) {
   // ... re-reserve in the same spot multiple areas...
   for (int stripe = 0; stripe < num_stripes; stripe++) {
     address q = p + (stripe * stripe_len);
-    q = (address)os::attempt_reserve_memory_at((char*)q, stripe_len);
+    q = (address)os::attempt_reserve_memory_at((char*)q, stripe_len, true);
```

Comment:
A comment here would hurt no one.

Reply:
I used the existing "executable" flag here, which already had a comment explaining it.